clang/test/CodeGenHLSL/builtins/RasterizerOrderedBuffer-annotations.hlsl

```diff
@@ -0,0 +1,20 @@
+// RUN: %clang_cc1 -triple dxil-pc-shadermodel6.0-compute -x hlsl -emit-llvm -disable-llvm-passes -o - %s | FileCheck %s
```

Comment:
I think this is fine for testing this change, but ROVs are not allowed to be used in CS (only PS).

This shader does compile successfully because they buffers are not used, but a use here will cause a validation error

https://godbolt.org/z/zEWr7bdsP

```
error: validation errors
error: RasterizerOrdered objects are only allowed in 5.0+ pixel shaders. 'Buffer1'
Validation failed.
```

Probably worth a comment in the test for this.

Reply:
That's a fair point. I went ahead and changed the triple to specify a pixel shader to avoid any confusion / future validation issues